templates/static/locales/it/translation.json

```diff
@@ -0,0 +1,54 @@
+{
+	"localiser": {
+		"credit": "Luke Granger-Brown"
```

Comment:
Do you want to fix this to be yourself? It's just a credit for the person who wrote the translation, so your name or handle :)